apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -2194,17 +2194,17 @@ export const extensions: Extensions = {
                     contentType: {
                         label: "Content Type",
                         placeholder: "JSON",
-                        hint: "The content type of the API request. Accepted value is JSON. (Default is JSON)"
+                        hint: "The content type of the API request. Accepted values are 'FORM' or 'JSON'"
```

Comment:
Please update the content in other languages as well. i.e french & sinhala.

Reply:
Done